atspi-common/src/events/mod.rs

```diff
@@ -542,6 +584,11 @@ impl TryFrom<&zbus::Message> for Event {
 				let ev = AddAccessibleEvent::try_from(msg)?;
 				Ok(Event::Cache(CacheEvents::Add(ev)))
 			}
+			// LegacyCacheAdd signature
+			"((so)(so)(so)a(so)assusau)" => {
```

Comment:
If you look at the top of this `TryFrom<Message> for Event` impl, you will see we are comparing to a 'Marshalled' signature, because that is what `body_signature()` returns. It has no outer parentheses, typically one pair less than our unmarshalled / Rust type ``Type::Signature()` ones.
In other words, I think we do not want the outer parentheses here, also goes for the other signatures here.